octobot/src/server/github_handler.rs

```diff
@@ -406,6 +407,25 @@ impl GithubEventHandler {
         let mut participants = pull_request.assignees();
         // add the author of the PR
         participants.push(pull_request.user().clone());
+        // add team members
+        let teams = pull_request.teams();
+        let futures = teams.iter().map(|t| {
+            self.github_session
+                .get_team_members(t.org(), t.slug.as_str())
+        });
+        let teams = join_all(futures).await;
+        let mut members: Vec<User> = vec![];
+        teams.iter().for_each(|r| {
+            let member = match r {
+                Ok(r) => r.clone(),
+                Err(e) => {
+                    error!("Error getting team members: {}", e);
+                    vec![]
+                }
+            };
+            members.append(&mut member.clone());
+        });
+        participants.append(&mut members);
```

Comment:
It's my first stab at rust and I'm sure this is very rough. Suggestion/fixup will be very much appreciated.

Reply:
well done for first stab! 🙌